libs/product/src/drivers/magento/transforms/product-transformer.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { DaffProductUnion } from '../../../models/product-union';
+import { DaffProductTransformerInterface } from '../../interfaces/product-transformer.interface';
+import { DaffProductMagentoDriverModule } from '../product-driver.module';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffMagentoProductTransformerService implements DaffProductTransformerInterface<DaffProductUnion> {
+
+  transform(response: any): DaffProductUnion {
+    const product = response.products.items[0];
+
+    return {
+      id: product.sku,
+      url: product.url_key,
+      name: product.name,
+      images: [
+        {url: product.image.url, id: "0", label: "My label"},
```

Comment:
Looks like this is a remnant that shouldn't be here anymore

Reply:
done.